engine/src/exec.h

```diff
@@ -52,6 +52,12 @@ enum MCExecValueType
 
 struct MCExecValue
 {
+public:
+	MCExecValue()
+	{
+		MCMemoryClear(*this);
```

Comment:
Ick. Looking forward to full C++11 support (where you can add constructors to unions)...